src/modules/apps/undeprecate.ts

```diff
@@ -68,6 +68,8 @@ const prepareUndeprecate = async (appsList: string[]): Promise<void> => {
       }
     }
   }
+
+  await returnToPreviousAccount({ previousAccount: originalAccount, previousWorkspace: originalWorkspace })
```

Comment:
I noticed that we have a call to `returnToPreviousAccount()` at the end of the `undeprecate` command, but we do not have it for the `deprecate`. Is that on purpose?
With this difference, for the `deprecate` command we'll not return to the previous account either when the status is 404 or when it's successful, but we will do it for the `undeprecate`.

If I remember correctly, we want to return to the previous account independently of success or failure. Is that correct?
If that is the case, what do you think about adding a `finally` block to the `try/catch`? We could call the `returnToPreviousAccount()` there.